addons/mail/static/tests/mock_server/mock_models/discuss_channel.js

```diff
@@ -767,6 +767,7 @@ export class DiscussChannel extends models.ServerModel {
         this.write([id], {
             last_interest_dt: serializeDateTime(today()),
         });
+        delete kwargs.special_mentions;
```

Comment:
Can't you write the full code here (1 extra line pretty much) so that `@everyone` is supported in mock server too?